code/game/gamemodes/vampire/traitor_vamp.dm

```diff
@@ -0,0 +1,45 @@
+/datum/game_mode/traitor/vampire
+	name = "traitor+vampire"
+	config_tag = "traitorvamp"
+	traitors_possible = 3 //hard limit on traitors if scaling is turned off
+	restricted_jobs = list("AI", "Cyborg")
+	required_players = 0
+	required_players_secret = 0
```

Comment:
Also here too, pretty sure the game mode would be terrible with only 1 or 2 players.